### PR TITLE
Closes #1606 - Change "Url" text to "URL"

### LINKED ISSUE #1606: "URL" is capitalized as "Url"
- GitLens Version: v11.6.0
- Git Version: 2.31.1

- VSCode Version: 1.59.0
- OS Version: macOS 11.5.2

Steps to Reproduce:

1. Invoke the Command Palette
2. Observe the option "GitLens: Copy Remote Commit Url"

As an acronym, [URL](https://en.wikipedia.org/wiki/URL) is almost always capitalized including in the [Visual Studio Code Language Packs](https://github.com/microsoft/vscode-loc).

### PR DESCRIPTION
**Pull request by pguilbert**
---
#1606

# Description

Change all occurrence of the capitallized "Url" to "URL" to match with the [Visual Studio Code Language Packs](https://github.com/microsoft/vscode-loc).

# Checklist

- [x] I have followed the guidelines in the Contributing document
- [x] My changes follow the coding style of this project
- [x] My changes build without any errors or warnings
- [x] My changes have been formatted and linted
- [ ] My changes include any required corresponding changes to the documentation
- [x] My changes have been rebased and squashed to the minimal number (typically 1) of relevant commits
- [x] My changes have a descriptive commit message with a short title, including a `Fixes $XXX -` or `Closes #XXX -` prefix to auto-close the issue that your PR addresses

### CODE REVIEW

**Comment by pguilbert** [comment]
... I don't know how I missed the previous PR #1607. 

Is this issue still relevant ?

**Comment by JamieEdge** [comment]
I agree that the issue is still relevant. URL is an acronym and should be capitalised. The current capitalisation was noticeable to me and led me to find this pull request and its related issue.

Under certain symbol naming conventions it is not unusual to see only the first letter of the acronym capitalised on occasion to improve readability (e.g. `repositoryUrlLength`), but in a user interface this is not correct.

The [Oxford English Dictionary](https://www.oed.com/view/Entry/258858), [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/url) and [Merriam-Webster Dictionary](https://www.merriam-webster.com/dictionary/URL) all define URL as a fully capitalised acronym and use it as such in their examples.

File: src/commands/openBranchesOnRemote.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export class OpenBranchesOnRemoteCommand extends ActiveEditorCommand {
 		const repoPath = await getRepoPathOrActiveOrPrompt(
 			gitUri,
 			editor,
-			args?.clipboard ? 'Copy Remote Branches Url' : 'Open Branches on Remote',
+			args?.clipboard ? 'Copy Remote Branches URL' : 'Open Branches on Remote',
 		);
 		if (!repoPath) return;
```
Comment (seahyinghang8) [inline src/commands/openBranchesOnRemote.ts:55]: what is this

**Comment by dnicolson** [comment]
Thank you for your comment @JamieEdge and your contribution @pguilbert, I believe this issue is still relevant. It's unfortunate that https://github.com/gitkraken/vscode-gitlens/pull/1607 was closed.

**Comment by eamodio** [comment]
Thanks! I've come around and am open to merging this once the conflicts are resolved and the other missing instances are included.

**Review by eamodio — CHANGES_REQUESTED** [review]
See main comment

**Comment by dnicolson** [comment]
@pguilbert are you able take a look at this?

**Comment by dnicolson** [comment]
@eamodio please have a look at https://github.com/gitkraken/vscode-gitlens/pull/2341.

**Pull request closed without merging**